addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -83,12 +83,37 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
     // Private
     //--------------------------------------------------------------------------
 
+    _getAttachmentsDomain: function () {
+        var domain = ['|', ['id', 'in', _.pluck(this.attachments, 'id')]];
+        if (this.recordData.model) {
+            domain = domain.concat([
+                '&',
+                '&',
+                ['res_model', '=', this.model],
+                ['res_id', '=', this.res_id|0],
```

Comment:
`parseInt`?

Reply:
It's an integer or null, |0 is speeder and simple